### ISSUE TITLE
Remove extra 333fm scrambles

### ISSUE THREAD

**Issue by jfly** [thewca/worldcubeassociation.org#2992]
(cc-ing @SAuroux to get his thoughts on this)

TNoodle's new UI had a bug where it generated extra scrambles for 333fm that would show up in the results json and eventually get uploaded to the WCA website. See https://github.com/thewca/tnoodle/issues/358 for more information.

The bug is fixed over on TNoodle, and after a version of TNoodle is released with the fix, we may want to go through the database and remove the extra scrambles we know weren't used.

In terms of removing these unnecessary scrambles, I ran the following query against our production database:

```
mysql> SELECT DISTINCT competitionId FROM Scrambles WHERE eventId="333fm" AND isExtra=1;
+-----------------------------+
| competitionId               |
+-----------------------------+
| AboardthePyraminxTrain2018  |
| AftabCubingDays2018         |
| AnnuntiaOpen2018            |
| ArcticOpen2018              |
| ArnoldClassicAfrica2018     |
| BairesCentroOpen2018        |
| Balakan2018                 |
| BandungZamanNow2018         |
| BanjaLukaOpen2018           |
| BerkeleySpring2018          |
| BerkeleyWinter2018          |
| BigCubeBonanza2018          |
| BrasiliaOpen2018            |
| BrilliantBNBB2018           |
| CarteBlancheKubeOpen2018    |
| CCS2018                     |
| CebuOpen2016                |
| ChakchakcubinginKazan2018   |
| ChennaiMindOverMatter2018   |
| CMITessellateCubeOpen2018   |
| CubeFactoryLeagueLuty2018   |
| DallasFortWorth2018         |
| DesafiodeVerao2018          |
| DneprNeverFails2018         |
| DontPeek2018                |
| DunasOpen2018               |
| ElEscorialOpen2018          |
| ElsassOpen2018              |
| EretumOpen2018              |
| FariaBritoOpen2018          |
| FinnishChampionship2018     |
| FinnishOpen2018             |
| FMCEurope2018               |
| France2018                  |
| FrankfurtCubeDays2018       |
| GermanOpen2018              |
| GLSCupII2018                |
| HalifaxSpring2018           |
| HeimaeyOpen2018             |
| HessenOpen2018              |
| ImaginaliaOpen2018          |
| IMineirimOpen2018           |
| Indiana2014                 |
| Indiana2015                 |
| IndianNationals2018         |
| IndianOpen2018              |
| Israelspecial2018           |
| KansaiOpen2018              |
| KeepPortlandQuiet2018       |
| KrasnoyarskWinterOpen2018   |
| LesFranquesesOpen2018       |
| LierOpen2018                |
| LSD2018                     |
| MatoGrandeOpen2018          |
| Meadowlands2018             |
| MedanOpen2018               |
| MentalBreakdownGermany2018  |
| MetroSpeedcubingOpenIII2018 |
| MichiganCubingClubBeta2018  |
| MindGames2018               |
| MinnesotaCubeMelt2018       |
| MoreThanOneCube2018         |
| Murcia2018                  |
| NacionalesUruguayasUCHI2018 |
| NoPlaceForDNFSpring2018     |
| NorwegianChampionship2018   |
| OpenBerry2018               |
| PaodeQueijo2018             |
| PBQWaterloo2018             |
| PoliMiOp3n2018              |
| PorelHonor2018              |
| PortoAlegreOpen2014         |
| PueblaBeQuiet2018           |
| Pulmapelipaivat2018         |
| RigaSpecial2018             |
| RockvilleQualifier2018      |
| RovigoOpen2018              |
| SchwandorfOpen2018          |
| SDWinter2018                |
| SESCGravatai2018            |
| ShanghaiLoveinECNU2018      |
| SidesBahia2018              |
| SilencioTotal2018           |
| SNUOpen2018                 |
| SouthernChampionship2018    |
| SPOpen2014                  |
| SpringCubeInMinsk2018       |
| SRMCubeOpen2018             |
| StenayOpen2018              |
| SwedishChampionship2018     |
| SwisscubingCupI2018         |
| TapiolaWinterOpen2018       |
| ThreeSecondsSnakes2018      |
| Tlaxcala2018                |
| UberlandiaSpecials2018      |
| UkrainianOpen2018           |
| VilniusOpen2018             |
| VisayasChampionship2018     |
| VoronezhOpen2018            |
| WarsawCubeMasters2018       |
| WesternChampionship2018     |
| ZurichOpen2018              |
+-----------------------------+
102 rows in set (3.15 sec)
```

You can see that there are some 2014, 2015 competitions with extra scrambles for 333fm. We probably don't want to delete those scrambles.

**Comment by SAuroux** [comment]
Thanks Jeremy. WRT will handle this as soon as a new TNoodle version is released.

**Comment by Baiqiang** [comment]
Just a note that we should be careful because there's real extra scramble in recent competitions like Indian Nationals 2018.

**Comment by SAuroux** [comment]
`SELECT competitionId, COUNT(*) as num FROM Scrambles WHERE competitionId LIKE "%2018" AND eventId='333fm' AND isExtra=1 GROUP BY competitionId HAVING num != 2`

returns 7 competitions that I looked at individually. 5 of them have 4 extras as they simply had 2 rounds of FMC. And then there are IndianNationals2018 and MedanOpen2018 with one extra scramble each. We know that IndianNationals2018 is legit, but I couldn't find anything about MedanOpen2018.

For simplicity, I decided to go ahead with:

`DELETE FROM Scrambles WHERE competitionId LIKE "%2018" AND eventId='333fm' AND isExtra=1 AND competitionId NOT IN ("IndianNationals2018", "MedanOpen2018")`

which deleted 208 extra scrambles (of course I tested with SELECT * first). I will still ask the delegate of MedanOpen2018 if an extra scramble was used in FMC or not and remove it later if appropriate.

In any case, the issue is mainly resolved, so I am closing this.
